include/mgba/internal/gba/hardware-extensions-ids.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021 Jeffrey Pfau
```

Comment:
```suggestion
/* Copyright (c) 2013-2021 Jeffrey Pfau
```

This file should probably be put in `extra` and merged with the file below.

Reply:
Fixed on all three files.